.github/workflows/ci.yml

```
@@ -0,0 +1,158 @@
+        path: ${{ env.R_LIBS }}
+        restore-keys: |
+          pkgcache-
+    - name: cache .doc
+      uses: actions/cache@v1
+      with:
+        key: doc-${{ github.sha }}
+        path: .doc
+    - name: cache .install
+      uses: actions/cache@v1
+      with:
+        key: install-${{ github.sha }}
+        path: .install
+    - name: build
+      run: make -j1
+      env:
+        NCPUS: 2
+        CI: true
+    - name: check for out-of-date Rd files
+      uses: infotroph/tree-is-clean@v1
+
+  test:
+    needs: build
+    runs-on: ubuntu-latest
+    container: infotroph/pecan-depends:latest
+    services:
+      postgres:
+        image: mdillon/postgis:9.5
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    env:
+      NCPUS: 2
+      PGHOST: postgres
+      CI: true
+    steps:
+    - uses: actions/checkout@v2
+    - name: install utils
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y openssh-client postgresql-client curl
+    - name: db setup
+      run: scripts/setup_testing_db.sh
+    - run: mkdir -p "${HOME}${R_LIBS#'~'}"
+    - name: cache R packages
+      uses: actions/cache@v1
+      with:
+        key: pkgcache-${{ github.sha }}
+        path: ${{ env.R_LIBS }}
+        restore-keys: |
+          pkgcache-
+    - name: cache .doc
+      uses: actions/cache@v1
+      with:
+        key: doc-${{ github.sha }}
+        path: .doc
+    - name: cache .install
+      uses: actions/cache@v1
+      with:
+        key: install-${{ github.sha }}
+        path: .install
+    - name: test
+      run: make test
+
+  check:
+    needs: build
+    runs-on: ubuntu-latest
+    container: infotroph/pecan-depends:latest
+    env:
+      NCPUS: 2
+      CI: true
+    steps:
+    - uses: actions/checkout@v2
+    - name: install ssh
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y openssh-client qpdf
+    - run: mkdir -p "${HOME}${R_LIBS#'~'}"
+    - name: cache R packages
+      uses: actions/cache@v1
+      with:
+        key: pkgcache-${{ github.sha }}
+        path: ${{ env.R_LIBS }}
+        restore-keys: |
+          pkgcache-
+    - name: cache .doc
+      uses: actions/cache@v1
+      with:
+        key: doc-${{ github.sha }}
+        path: .doc
+    - name: cache .install
+      uses: actions/cache@v1
+      with:
+        key: install-${{ github.sha }}
+        path: .install
+    - name: check
+      run: make check
+      env:
+        REBUILD_DOCS: "FALSE"
+        RUN_TESTS: "FALSE"
+
+  sipnet:
+    needs: build
+    runs-on: ubuntu-latest
+    container: infotroph/pecan-depends:latest
+    services:
+      postgres:
+        image: mdillon/postgis:9.5
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    env:
+      PGHOST: postgres
+    steps:
+    - uses: actions/checkout@v2
+    - run: sudo apt-get update && sudo apt-get install -y curl postgresql-client
```

We must follow the literal style 
```suggestion
    - run: |
        sudo apt-get update
        sudo apt-get install -y curl postgresql-client
```

---

Are there subtleties in behavior I should know here or is this purely a style thing? 

If it's just style: A good rule in general and If this line were even a few characters longer I'd agree. But I think when the command is conceptually one step (in this case we only want to update because we need it before installing) and fits comfortably on one line, I prefer `&&`.

---

Yaa... It's just for styling... I made these remarks as above in the code you used the literal style. But I will go with you because using `&&` is a better option as the second command will only be executed if the update returns exit status zero.